python/servo/command_base.py

```diff
@@ -50,12 +50,16 @@ def setlocale(name):
 
 def find_dep_path_newest(package, bin_path):
     deps_path = path.join(path.split(bin_path)[0], "build")
+    candidates = []
     with cd(deps_path):
-        candidates = glob(package + '-*')
-    candidates = (path.join(deps_path, c) for c in candidates)
-    candidate_times = sorted(((path.getmtime(c), c) for c in candidates), reverse=True)
-    if len(candidate_times) > 0:
-        return candidate_times[0][1]
+        for c in glob(package + '-*'):
+            candidate_path = path.join(deps_path, c)
+            candidate_output = path.join(candidate_path, "output")
+            if path.exists(candidate_output):
```

Comment:
Like I said above, this won't fix the problem of subdirs. I think we should rebase this upon #14601